### PR TITLE
grt: Converting TreeNode and TreeEdge pointers to vectors

### PR DESCRIPTION
**Pull request by luis201420**

### CODE REVIEW

File: src/grt/src/fastroute/include/DataType.h
Comment (eder-matheus) [inline src/grt/src/fastroute/include/DataType.h:173]: Use snake_case for member names: `nbr_count`

File: src/grt/src/fastroute/src/maze3D.cpp
Comment (eder-matheus) [inline src/grt/src/fastroute/src/maze3D.cpp:899]: Why they're losing const? Here and below.

**Review by eder-matheus — CHANGES_REQUESTED** [review]

File: src/grt/src/fastroute/src/maze3D.cpp
Comment (luis201420) [reply src/grt/src/fastroute/src/maze3D.cpp:899]: Because those variables are then modified in the function

Comment (eder-matheus) [reply src/grt/src/fastroute/src/maze3D.cpp:899]: I didn't notice some function calls that use these variables were marked as non-const.

**Review by eder-matheus — APPROVED** [review]
Waiting for secure-ci before merging.

**Pull request merged**
